homeassistant/components/vacuum/neato.py

```diff
@@ -24,7 +24,7 @@
 SCAN_INTERVAL = timedelta(minutes=5)
 
 SUPPORT_NEATO = SUPPORT_BATTERY | SUPPORT_PAUSE | SUPPORT_RETURN_HOME | \
-                 SUPPORT_STOP | SUPPORT_START | \
+                 SUPPORT_STOP | SUPPORT_START | SUPPORT_CLEAN_SPOT \
```

Comment:
There is no bitwise or operator after `SUPPORT_CLEAN_SPOT` before the backslash. Sorry that I missed it earlier.

Reply:
No, sorry for missing it. My original code had SUPPORT_CLEAN_SPOT on the last line, but I found the formatting to be more balanced with it on the middle line...

Done!